task_12/src/css/desktop/main.css

```diff
@@ -0,0 +1,64 @@
+main {
+  padding: 40px 30px 0;
+  font-family: "Ubuntu", sans-serif; }
+
+.button {
+  position: relative;
+  display: block;
+  overflow: visible;
+  /* IE9 & 10 */
+  max-width: 200px;
+  cursor: pointer;
+  transition: 500ms linear;
+  color: #fef1e0;
+  border: 1px solid #fef1e0;
+  border-radius: 5px;
+  outline: none;
+  background-color: #c29c5b;
+  box-shadow: inset 1px 1px 0 rgba(255, 255, 255, 0.25), inset 0 0 6px #a23227, inset 0 80px 80px -40px #ac3223, 1px 1px 3px rgba(0, 0, 0, 0.75);
+  /* shadow */
+  text-shadow: 0 1px 2px rgba(0, 0, 0, 0.75);
+  font-size: 14px; }
+  .button:hover {
+    background: #a61715;
+    box-shadow: inset 1px 1px 0 rgba(255, 255, 255, 0.25), inset 0 0 6px #da3b2c, inset 0 80px 80px -40px #dd4330, 1px 1px 3px rgba(0, 0, 0, 0.75);
+    /* shadow */
+    text-shadow: 0 1px 2px rgba(0, 0, 0, 0.75), 0 0 40px #fff; }
+  .button.button__add {
```

Comment:
.

Reply:
And here 

<img width="784" alt="screen shot 2018-02-05 at 15 39 33" src="https://user-images.githubusercontent.com/24464487/35807221-cacc515c-0a8a-11e8-93e3-144c66a41c9f.png">

Reply:
Actually, it does not look pretty good. If you write multiline properties (`box-shadow` in your case), add indent to values on following lines to visually group it with current pproperty separate it from other ones.

Also, you can use `//` to comment everything to the end of line.

Reply:
corrected